pingcap/tidb
# *: make transaction duration metrics more reasonable

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve? 
The transaction duration metrics seem not right for the users. The users may want an end to end duration.

### What is changed and how it works?
Make it more reasonable.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. run some workload
1. check out the transaction duration metrics

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the `tidb-ansible` repository
